demos/gray_scott.py

```diff
@@ -159,18 +159,12 @@ def qoi():
 
 # Finally, plot the outputs to be viewed in Paraview. ::
 
-ic = mesh_seq.get_initial_condition()
-for field, sols in solutions.items():
-    fwd_outfile = VTKFile(f"gray_scott/{field}_forward.pvd")
-    adj_outfile = VTKFile(f"gray_scott/{field}_adjoint.pvd")
-    fwd_outfile.write(*ic[field].subfunctions)
-    for i in range(num_subintervals):
-        for sol in sols["forward"][i]:
-            fwd_outfile.write(*sol.subfunctions)
-        for sol in sols["adjoint"][i]:
-            adj_outfile.write(*sol.subfunctions)
-    adj_end = Function(ic[field]).assign(0.0)
-    adj_outfile.write(*adj_end.subfunctions)
+solutions.export(
+    "gray_scott/forward.pvd",
```

Comment:
I had separate `forward.pvd` and `adjoint.pvd` files because the forward field has the initial condition field and adjoint doesn't - and VTU doesn't allow non-matching number of timesteps.

But in 6f96c365fbe6d631d4358b25d97fe0e7e6dc5e89 I made it so that we export fields with `NaN` for non-forward fields. So now we only have one file `solutions.pvd` with both fields :)

Reply:
Thanks, this makes sense but should definitely be documented in the docstring for the `export` method because it could lead to confusion.

Reply:
Done in eb82642805bd24068d7b2e5ba150ff351fbfb96c